src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/13.diff.sql

```diff
@@ -0,0 +1,540 @@
+/*************************************************************
+    Capturing changes for event
+**************************************************************/
+
+/*************************************************************
+    Resource change data table
+**************************************************************/
+
+IF NOT EXISTS (
+    SELECT 1 
+    FROM sys.tables
+    WHERE name = 'ResourceChangeData')
+BEGIN
+    CREATE TABLE dbo.ResourceChangeData
+    (
+        Id bigint IDENTITY(1,1) NOT NULL,
+        Timestamp datetime2(7) NOT NULL,
+        ResourceId varchar(64) NOT NULL,
+        ResourceTypeId smallint NOT NULL,
+        ResourceVersion int NOT NULL,
+        ResourceChangeTypeId smallint NOT NULL
+    PRIMARY KEY CLUSTERED 
+    (
+        [Id] ASC
+    )WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON, OPTIMIZE_FOR_SEQUENTIAL_KEY = OFF) ON [PRIMARY]
+    ) ON [PRIMARY]
+
+    ALTER TABLE dbo.ResourceChangeData ADD  CONSTRAINT DF_ResourceChangeData_Timestamp  DEFAULT (sysutcdatetime()) FOR Timestamp
+END
+GO
+
+IF EXISTS (
+  select 1 from information_schema.columns where table_name = 'ResourceChangeData'
+and column_name = 'Timestamp'
+and Table_schema = 'dbo'
+and column_default is NULL)
+BEGIN
+	 ALTER TABLE dbo.ResourceChangeData ADD  CONSTRAINT DF_ResourceChangeData_Timestamp  DEFAULT (sysutcdatetime()) FOR Timestamp
+END
+GO
+
+
+/*************************************************************
+    Resource change type table
+**************************************************************/
+
+IF NOT EXISTS (
+    SELECT 1 
+    FROM sys.tables
+    WHERE name = 'ResourceChangeType')
+BEGIN
+
+	 CREATE TABLE dbo.ResourceChangeType(
+		 ResourceChangeTypeId smallint IDENTITY(0,1) NOT NULL,
+		 Name nvarchar(50) NOT NULL,
```

Comment:
Also unique?

Reply:
ResourceChangeTypeId is a primary key that guarantees uniqueness.

Reply:
Sorry, I should be more clear, the comment was pointing to the "Name".

Reply:
I see. No, the name is not unique.

Reply:
I thought we could make it unique to make it less error prone.

Reply:
Added a unique constraint.